service-base/src/test/java/org/eclipse/hono/service/AbstractProtocolAdapterBaseTest.java

```diff
@@ -370,7 +370,7 @@ public void testGetRegistrationAssertionSucceedsForExistingDevice(final VertxTes
 
         final Checkpoint assertion = ctx.checkpoint();
         // WHEN an assertion for the device is retrieved
-        adapter.getRegistrationAssertion("tenant", "device", null)
+        adapter.getRegistrationAssertion("tenant", "device", null, mock(SpanContext.class))
```

Comment:
this method is being invoked intentionally here. We are testing both variants, the deprecated one, and the not-deprecated one (few lines below) ...